packages/shared/routes/dashboard/Dashboard.svelte

```diff
@@ -24,7 +24,6 @@
     const tabs = {
         wallet: Wallet,
         settings: Settings,
-        staking: Staking,
         developer: Developer,
```

Comment:
Might be nice to have something like:
```typescript
enum AppContext {
    Developer = 'developer',
    Onboarding = 'onboarding'
    Wallet = 'wallet',
}

const tabs = {
    [AppContext.Wallet]: Wallet,
    ...
}
```

This would also work with the `en.json` locale if we add more hierarchy to the `views`, but that's definitely for another day.

Reply:
True, but let's keep it for a separate PR, since I only removed this due to warnings that staking was missing.